packages/financial-templates-lib/test/truffle/BalancerPriceFeed.js

```diff
@@ -157,6 +157,31 @@ contract("BalancerPriceFeed.js", function(accounts) {
     assert.equal(dexPriceFeed.getCurrentPrice(), null);
   });
 
+  it("One event within window, several before", async function() {
```

Comment:
Unrelated to the core of this PR, but I noticed that @mrice32 never added tests specific to #2465 . Do you agree that these tests correctly test changes introduced in that PR?

Reply:
Yeah these tests actually indicate the balancer price feed doesn't have the same problem that uniswap did (until your PR fixed it)!